PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalInternalClient.kt

```diff
@@ -50,12 +50,28 @@ internal class PayPalInternalClient(
                     payPalRequest.enablePayPalAppSwitch = isPayPalInstalled(context)
                 }
 
+                val returnLinkType = getReturnLinkTypeUseCase()
+                val navigationLink = when (returnLinkType) {
+                    GetReturnLinkTypeUseCase.ReturnLinkType.APP_LINK -> merchantRepository.appLinkReturnUri
+                    GetReturnLinkTypeUseCase.ReturnLinkType.DEEP_LINK -> merchantRepository.returnUrlScheme
```

Comment:
Same comment here - should we be checking that the merchant has included a returnUrlScheme? If a merchant only wants to support app links and app links are not enabled on the device we should have an error scenario I assume

Reply:
Similar to the comment above, we're defaulting the returnUrlScheme to the merchant's package name. Did we want to change this behavior?